packages/angular/src/builders/webpack-browser/schema.json

```diff
@@ -344,9 +344,14 @@
         "type": "string"
       },
       "default": []
+    },
+    "customBuilder": {
```

Comment:
In the resulting `angular.json`/`workspace.json` file, this will look like:
```json
"builder": "@nrwl/angular:webpack-browser",
"customBuilder": "@angular-builders/custom-webpack"
```
Which is a little confusing. It would be more accurate to give this a name that represents that our builder is handing it off to the third party builder.

Something along the lines of `targetBuilder` perhaps.

Reply:
It should be good to go now.